bin/minikube/launch.py

```diff
@@ -0,0 +1,1049 @@
+        "-c",
+        f"create database {config.db_name}",
+    ]
+
+    typer.echo("Initializing database...")
+    if is_debug:
+        typer.echo(" ".join(create_database))
+
+    def initialize_database(
+        *, is_debug: bool = False, nb_attempts_left: Optional[int] = None
+    ) -> None:
+        should_print_stderr = nb_attempts_left == 0 if nb_attempts_left else False
+        result = subprocess.run(
+            create_database,
+            stdout=subprocess.DEVNULL if not is_debug else None,
+            stderr=subprocess.DEVNULL if not should_print_stderr else subprocess.PIPE,
+        )
+
+        return_code = result.returncode
+
+        if return_code == 0:
+            raise StopIteration
+
+        if should_print_stderr:
+            typer.echo(result.stderr)
+
+    do_retry(
+        initialize_database,
+        on_success_message="Database setup!",
+        on_failure_message="Could not initialize database",
+        sleep_for_seconds=15,
+        is_debug=is_debug,
+    )
+
+    postgres_host = f"{postgres_release}-postgresql.{namespace}.svc.cluster.local"
+    postgres_result = PostgresResult(
+        release_name=release_name,
+        namespace=namespace,
+        db_host=postgres_host,
+        db_password=config.db_password,
+        db_name=config.db_name,
+        db_user=config.db_name,
+    )
+
+    save_result(postgres_result)
+    return postgres_result
+
+
+@app.command()
+def create_redis(
+    release_name: str = typer.Option(..., envvar="RELEASE_NAME"),
+    namespace: str = typer.Option(..., envvar="NAMESPACE"),
+    is_debug: bool = typer.Option(False, "--debug", help="Show verbose output."),
+) -> RedisResult:
+    """Create a redis message queue setup to run the Avatar API"""
+
+    if not is_minikube_running():
+        typer.echo("Minikube must be running. Run with 'minikube start'.")
+        raise typer.Abort()
+
+    redis_release = f"{release_name}-redis"
+    existing_redis_release = subprocess.check_output(
+        ["helm", "list", "--namespace", namespace, "--filter", redis_release, "-q"],
+        text=True,
+    ).strip()
+
+    if existing_redis_release == redis_release:
+        typer.echo("A redis release already exists in that namespace.")
+        raise typer.Abort()
+
+    flags = ["--debug", "--create-namespace"]
+    namespace_command = ["--namespace", namespace]
+    values = ["--set", "auth.enabled=false"]
+    install_redis = [
+        "helm",
+        "install",
+        redis_release,
+        "bitnami/redis",
+        *values,
+        *namespace_command,
+        *flags,
+    ]
+
+    typer.echo("Creating redis Helm release...")
+    if is_debug:
+        typer.echo(" ".join(install_redis))
+
+    result = subprocess.run(
+        install_redis,
+        stdout=subprocess.DEVNULL if not is_debug else None,
+        stderr=subprocess.PIPE,
+    )
+
+    if result.returncode != 0:
+        typer.echo(result.stderr)
+        typer.echo("Could not initialize Redis :(")
```

Comment:
Inverse those 2 lines

Reply:
The custom message would be lost in the `stderr` output, I found it helpful for the output to be on the last line as it's the first thing you see when it fails